spk/activebackup-prometheus-exporter/src/service-setup.sh

```diff
@@ -0,0 +1,5 @@
+AB_EXPORTER="${SYNOPKG_PKGDEST}/bin/activebackup-prometheus-exporter"
+SERVICE_COMMAND="${AB_EXPORTER}"
+SERVICE_PORT=9763
```

Comment:
Both `SERVICE_PORT` and `SERVICE_PORT_TITLE` are Makefile variables https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables
Please set them in spk/activebackup-prometheus-exporter/Makefile so that port can be configured from network firewall administration

Reply:
Fixed :)